profile/README.md

```diff
@@ -8,18 +7,18 @@ Welcome to the Configit SDK, home of libraries and sample code for Configit part
 
```

Comment:
I don't know why I can't make a suggestion comment here but I would move the whole second sentence onto the next line if possible. 

> **Getting access**: The Configit SDK repositories and NuGet/npm packages are private. 
> To request access, mail configit-sdk-req@configit.com.